api/lib/domain/usecases/create-sessions.js

```diff
@@ -37,11 +37,17 @@ module.exports = async function createSessions({
       const savedSession = await sessionRepository.save(domainSession, domainTransaction);
 
       await bluebird.mapSeries(domainSession.certificationCandidates, async (certificationCandidate) => {
+        const billingMode = CertificationCandidate.translateBillingMode(certificationCandidate.billingMode);
         const domainCertificationCandidate = new CertificationCandidate({
           ...certificationCandidate,
           sessionId: savedSession.id,
+          billingMode,
         });
 
+        if (domainCertificationCandidate.billingMode === 'FREE') {
```

Comment:
```suggestion
        if (domainCertificationCandidate.billingMode === BILLING_MODES.FREE || domainCertificationCandidate.billingMode === BILLING_MODES.PAID) {
```
Mais ce devrait peut etre etre fait dans le constructeur

Reply:
Code temporaire